Vagrantfile

```diff
@@ -0,0 +1,134 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# Simulate ActiveModel::Type::Boolean.new.cast(value)
+$FALSE_VALUES = [false, 0, "0", "f", "F", "false", "FALSE", "off", "OFF"]
+
+# Should Vagrant modify /etc/exports automatically?
+$nfs_export = $FALSE_VALUES.none? ENV['VAGRANT_NFS_EXPORT']
+
+prov_env = {
+  "DEBIAN_FRONTEND" => "noninteractive",
+}
+
+$init = <<SCRIPT
+  set -e
+  apt -y update
+  apt -y install \
+    build-essential git tmux vim \
+    autoconf automake libtool \
+    libssl-dev net-tools
+SCRIPT
+
+$python = <<SCRIPT
+  set -e
+  apt -y install \
+    python3-all python3-dev \
+    python3-pip python3-venv
+SCRIPT
+
+$docker_ce = <<SCRIPT
+  set -e
+  apt -y install \
+    apt-transport-https \
+    ca-certificates \
+    curl \
+    software-properties-common
+  curl -fsSL https://download.docker.com/linux/ubuntu/gpg |
+    apt-key add -
+  add-apt-repository \
+    "deb [arch=amd64] https://download.docker.com/linux/ubuntu \
+    $(lsb_release -cs) \
+    stable"
+  apt -y install \
+    containerd.io \
+    docker-ce \
+    docker-ce-cli
+SCRIPT
+
+$docker_compose = <<SCRIPT
+  set -e
+  curl \
+    -L "https://github.com/docker/compose/releases/download/1.29.2/docker-compose-$(uname -s)-$(uname -m)" \
+    -o /usr/local/bin/docker-compose
+  chmod +x /usr/local/bin/docker-compose
+SCRIPT
+
+$mininet = <<SCRIPT
+  set -e
+  apt -y install \
+    mininet \
+    openvswitch-switch
+SCRIPT
+
+# WAND Open vSwitch
+$wand_ovs = <<SCRIPT
+  set -e
+  apt install apt-transport-https
+  echo "deb https://packages.wand.net.nz $(lsb_release -sc) main" >/etc/apt/sources.list.d/wand.list
```

Comment:
This apt repo is now deprecated, try our new shiny OVS/DPDK repo that we published a couple months back :)

```
sudo apt-get install apt-transport-https curl lsb-release
sudo curl -1sLf https://dl.cloudsmith.io/public/wand/openvswitch/gpg.2E801E8CCE233F26.key -o /etc/apt/trusted.gpg.d/wand-openvswitch.asc
curl -1sLf "https://dl.cloudsmith.io/public/wand/openvswitch/config.deb.txt?distro=$(lsb_release -is)&codename=$(lsb_release -sc)" | sudo tee /etc/apt/sources.list.d/wand-openvswitch.list
sudo apt-get update
```

Reply:
Thanks for the pointer - I'll update